applications/app/templates/serviceWorker.scala.js

```diff
@@ -23,11 +20,11 @@ var fetchAll = function (inputs) {
     }));
 };
 
-var cachePageAndAssetResponses = function (jsonResponse, assetResponses) {
```

Comment:
Maybe you could name space all of the offline crossword code by wrapping them in an IIFE? It would be good to group the notifications stuff in the same way!

``` js
// Offline crossword
(() => {
  var updateCache = () => {};
})();
```